### ISSUE TITLE
Better exception handling when STRUCT is empty

### ISSUE THREAD

**Issue by subkanthi** [Altinity/clickhouse-sink-connector#253]
```
478811 2023-05-26 14:09:38.624 [main] ERROR com.altinity.clickhouse.debezium.embedded.cdc.DebeziumChangeEventCapture  - Exception processing record
java.lang.NullPointerException
478811 2023-05-26 14:09:38.624 [main] ERROR com.altinity.clickhouse.debezium.embedded.cdc.DebeziumChangeEventCapture  - STRUCT EMPTY
478811 2023-05-26 14:09:38.624 [main] ERROR com.altinity.clickhouse.debezium.embedded.cdc.DebeziumChangeEventCapture  - Exception processing record
java.lang.NullPointerException
478811 2023-05-26 14:09:38.624 [main] ERROR com.altinity.clickhouse.debezium.embedded.cdc.DebeziumChangeEventCapture  - STRUCT EMPTY
478811 2023-05-26 14:09:38.624 [main] ERROR com.altinity.clickhouse.debezium.embedded.cdc.DebeziumChangeEventCapture  - Exception processing record
```

**Comment by subkanthi** [comment]
```
STRUCT EMPTY + Record(EmbeddedEngineChangeEvent [key=null, value=SourceRecord{sourcePartition={server=embeddedconnector}, sourceOffset={transaction_id=null, ts_sec=1698354829, file=mysql-bin-changelog.010552, pos=31116708, gtids=10f621a7-f3ba-11ea-8017-02015778a3f9:1-2181437829,2c754c71-539c-11ee-bdd4-129c4c0ae8a7:1-160,4180c625-be60-11e8-8d56-a4bf010a41be:3652387912-3652388179:3652388182-4169697097,6d1e5f7e-b5c9-11e8-a514-a4bf011babaa:16049745253-16778132426, row=1, server_id=764132202, event=2}} ConnectRecord{topic='embeddedconnector.', kafkaPartition=null, key=Struct{entry_id=190517234}, keySchema=Schema{embeddedconnector...Key:STRUCT}, value=null, valueSchema=null, timestamp=null, headers=ConnectHeaders(headers=)}, sourceRecord=SourceRecord{sourcePartition={server=embeddedconnector}, sourceOffset={transaction_id=null, ts_sec=1698354829, file=mysql-bin-changelog.010552, pos=31116708, gtids=10f621a7-f3ba-11ea-8017-02015778a3f9:1-2181437829,2c754c71-539c-11ee-bdd4-129c4c0ae8a7:1-160,4180c625-be60-11e8-8d56-a4bf010a41be:3652387912-3652388179:3652388182-4169697097,6d1e5f7e-b5c9-11e8-a514-a4bf011babaa:16049745253-16778132426, row=1, server_id=764132202, event=2}} ConnectRecord{topic='embeddedconnector..', kafkaPartition=null, key=Struct{entry_id=190517234}, keySchema=Schema{embeddedconnector..Key:STRUCT}, value=null, valueSchema=null, timestamp=null, headers=ConnectHeaders(headers=)}])
```

**Comment by subkanthi** [comment]
Changed to WARN and added log to print the full source record.
Fixed in #346
